pyrf/gui/gui.py

```diff
@@ -222,6 +231,31 @@ def start_playback(self, playback_filename):
         self.controller.set_device(playback_filename=playback_filename)
         self.show()
 
+    def save_configuration(self):
+        filename = time.strftime('config-%Y-%m-%d-%H%M%S')
+        names = glob.glob(filename + '*.config')
+
+        if (filename + '.config') in names:
+            count = names.count(filename)
+            filename += '(%d)' % count
+        filename += '.config'
```

Comment:
You might want to use a less generic config file name like ".pyrfconf" or something, but .config or .ini is fine too